frontend/src/pages/store/[store].svelte

```diff
@@ -42,6 +42,7 @@
 	let currentPage = 1;
 	let totalPages = 1;
 	const ITEMS_PER_PAGE = 12;
+	const LOW_STOCK = 5; //Low stock if there are this many items or less
```

Comment:
Why 5?

Reply:
I was assuming that most stores will usually only have a relatively small quantity of items, (I was thinking around 20 max), so 5 would be 25% of that.

Thinking about it now, should we let this be defined by the store owner? 
We could still have a default like 5

Reply:
Done